test/Factories.hs

```diff
@@ -37,8 +37,8 @@ createUser ident = do
     Entity planId _ <- createFreePlan
 
     insertEntity User
-        { userName = "John Smith (" ++ ident ++ ")"
-        , userEmail = "john-" ++ ident ++ "@gmail.com"
+        { userName = "Dummy Login"
+        , userEmail = "dummy@example.com"
```

Comment:
> I think there is something funny about the data coupling here

Definitely, I think any time you talk about a "dummy" login in your app, funny things may happen. The real fix would be an less hacky way to authenticate in tests.